ompi/mca/mtl/psm2/mtl_psm2.c

```diff
@@ -101,7 +101,7 @@ int ompi_mtl_psm2_module_init(int local_rank, int num_local_procs) {
     char env_string[256];
     int rc;
 
-    generated_key = getenv(OPAL_MCA_PREFIX"orte_precondition_transports");
+    generated_key = getenv("PMIX_MCA_opa_precondition_transports");
```

Comment:
I don't know why this is being prefixed with `PMIX_MCA_` as it really isn't a parameter that controls PMIx. Would it make more sense to prefix/name it something else, perhaps more OPA-specific?

Reply:
> I don't know why this is being prefixed with `PMIX_MCA_` as it really isn't a parameter that controls PMIx. 

The prefix is being attached by ``pmix_mca_base_var_env_name()``. I don't mind if we change it, but the prefix seems to be used entirely because it's PMIx that's generating the value.

Reply:
I understand _how_ the name gets that way - just not sure if we _want_ it to be that way as it isn't an MCA param that PMIx will recognize. Might confuse people who see it in their environment and think that PMIx should do something with it, or try to manually set it in an attempt to influence PMIx behavior.

Maybe just use `OPA_TRANSPORT_KEY`? Something that makes it clear it is solely intended for OPA's use?

Reply:
That works for me. OPA is the only remaining user of the "preconditioned transports" key, so it seems appropriate.